x-pack/plugins/reporting/server/routes/common/generate/request_handler.test.ts

```diff
@@ -246,47 +245,13 @@ describe('Handle request to generate', () => {
     });
 
     test('generates the download path', async () => {
-      const response = (await requestHandler.handleGenerateRequest(
```

Comment:
This test was not doing what the title says it does. Addressing that problem allowed the body of the test to be simplified.